node-data/src/bls.rs

```diff
@@ -178,66 +169,174 @@ fn read_from_file(
     hasher.update(pwd.as_bytes());
     let hashed_pwd = hasher.finalize().to_vec();
 
-    let bytes = match decrypt(&ciphertext[..], &hashed_pwd) {
-        Ok(bytes) => bytes,
-        Err(_) => {
-            let bytes = decrypt(&ciphertext[..], &hashed_pwd).map_err(|e| {
-                anyhow::anyhow!("Invalid consensus keys password {e}")
-            })?;
-            warn!("Your consensus keys are in the old format");
-            warn!("Consider to export them using a new version of the wallet");
```

Comment:
Did you remove the legacy support?
If so, please add it to the changelog

Reply:
There was no legacy support.